### DISCUSSION TITLE
api calls to openAI api aren't executed in vercel. Files in Pages/Api are downloaded but nothing is generated

### DISCUSSION THREAD

**Discussion by khdjn** [vercel/community#1721] [Help] [answered]
### Summary

My website is working correctly locally. After deployment in vercel, when I click generate to execute the api call to open AI API, two errors appear:
- when injecting the page: 404 page not found 
- In Vercel logs: 308 error and the request appears as "GET" not "POST"
The api folders is in pages folder and trailing Splash is deactivated. I use a sitemap generator for the website.

### Example

https://magiq.vercel.app

**Comment by leerob** [comment]
Hey there! Sorry about this. We have a bunch of templates that use OpenAI that you can get started with. All of the code is open source, so should hopefully help you debug! http://vercel.ai/

**Comment by khdjn** [comment, marked as answer]
eventually I had to change some configurations in my project for it to work and move the api folder outside of pages. Now it works!
